schemas/1.5/dbt_cloud-1.5.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "project-id": {
+      "type": "string"
```

Comment:
Are these actually strings and not numbers?

Reply:
as far as i can tell -- all the docs include quotes around the values